README.md

```diff
@@ -30,13 +294,59 @@ If you want to invoke a lambda specifically, here’s the example command:
 Otherwise, in the platform-forms-client, you just modify the ‘endpoint’ parameter of the LambdaClient to hit `http://127.0.0.1:3001` . I’ve done this through an environment variable:
 `LOCAL_LAMBDA_ENDPOINT=http://127.0.0.1:3001`
 
+You will also have to supply the following environment variables and values to platform-forms-client to get everything working with the local lambdas and localstack.
+
+```shell
+# localstack only simulates a us-east-1 region
+AWS_REGION=us-east-1
+LOCAL_S3_ENDPOINT=http://localhost:4566
```

Comment:
Update README.md here or in the client repository to add the RELIABILITY_FILE_STORAGE variable with the right value. It will be forms-local-reliability-file-storage if you have set up the Localstack environment.